extension.json

```diff
@@ -12,9 +12,9 @@
 	"license-name": "GPL-2.0-or-later",
 	"type": "semantic",
 	"requires": {
-		"MediaWiki": ">= 1.31",
+		"MediaWiki": ">= 1.39",
 		"extensions": {
-			"SemanticMediaWiki": ">= 3.0"
+			"SemanticMediaWiki": ">= 4.0"
```

Comment:
@JeroenDeDauw didn't we decide in another issue to remove SMW requirements from extension.json?

Reply:
I think that was in composer, removing the requirement to install. Not removing it from extension.json.

Reply:
@paladox yeah, you are right